code/datums/gamemodes/infestation.dm

```diff
@@ -33,6 +33,21 @@
 	for(var/i in GLOB.xeno_jelly_pod_turfs)
 		new /obj/structure/xeno/resin_jelly_pod(i, XENO_HIVE_NORMAL)
 
+	scale_weapons()
+
+/datum/game_mode/infestation/proc/scale_weapons(pop_override)
```

Comment:
autodoc. Although is there a reason this proc exists?

Reply:
I was suggested to have the sniper weapons scale with pop, though this wasn't a maint request. If it's better to just hard limit the numbers, I can do that instead

Reply:
no I mean, does this have a reason to exist as a separate proc? Its only used in one place.